packages/web/components/Form/FormWizard.js

```diff
@@ -257,26 +258,49 @@ const FormWizard = ({ steps, currentStep, onSubmit, onPrev, data, defaultValues,
 
 	const { technology: { revisions = [], status } = {} } = data;
 	const lastCuratorRevision = revisions.length ? revisions[revisions.length - 1] : null;
+	const renderStep = (step, index) => {
+		const showIcon = index < currentStepIndex || typeof step.icon !== 'undefined';
+		const Icon = index < currentStepIndex ? AiOutlineCheck : step.icon || null;
+		const showLink = index < currentStepIndex;
+		const isPublished = data.technology.status === 'published';
+		if (isPublished) {
+			return (
+				<StepItem completed={index <= currentStepIndex} key={step.slug}>
+					<Link href={`/technology/${data?.technology?.id}/edit/${step.slug}`}>
+						<div>
+							<StepNumber>{showIcon ? <Icon /> : index + 1}</StepNumber>
+							<StepLabel>{step.label}</StepLabel>
+						</div>
+					</Link>
+				</StepItem>
+			);
+		}
+
+		return (
+			<StepItem completed={index <= currentStepIndex} key={step.slug}>
+				{showLink ? (
+					<Link href={`/technology/${data?.technology?.id}/edit/${step.slug}`}>
+						<div>
+							<StepNumber>{showIcon ? <Icon /> : index + 1}</StepNumber>
+							<StepLabel>{step.label}</StepLabel>
+						</div>
+					</Link>
+				) : (
+					<div>
+						<StepNumber>{showIcon ? <Icon /> : index + 1}</StepNumber>
+						<StepLabel>{step.label}</StepLabel>
+					</div>
+				)}
+			</StepItem>
+		);
+	};
 
 	return (
 		<FormWizardContainer>
 			<StepsContainer>
 				<WebSteps>
-					{steps.map((step, i) => {
-						const showIcon = i < currentStepIndex || typeof step.icon !== 'undefined';
-						const Icon = i < currentStepIndex ? AiOutlineCheck : step.icon || null;
-						return (
-							<StepItem completed={i <= currentStepIndex} key={step.slug}>
-								<div>
-									<StepNumber>{showIcon ? <Icon /> : i + 1}</StepNumber>
-									<Link
-										href={`/technology/${data?.technology?.id}/edit/${step.slug}`}
-									>
-										<StepLabel>{step.label}</StepLabel>
-									</Link>
-								</div>
-							</StepItem>
-						);
+					{steps.map((step, index) => {
+						return renderStep(step, index);
 					})}
```

Comment:
Isso aqui pode ser feito assim também, é uma uma sintaxe mais simplifica quando não precisamos fazer nada dentro da função de callback
```suggestion
					{steps.map((step, index) => renderStep(step, index); )}
```